rero_ils/modules/apiharvester/utils.py

```diff
@@ -98,7 +98,7 @@ def get_records(url=None, name=None, from_date=None, max=0, size=100,
         request = requests.get(url)
         data = request.json()
 
-        total = data.get('hits', {}).get('total', 0)
+        total = data['hits']['total']['value']
```

Comment:
I will be raised `KeyError` if the request failed. Why not used  `data.get('hits', {}).get('total', {}).get('total', 0)` ?

Reply:
Normally a request has to have a ['hits']['total']['value'] value. I prefer to see the `KeyError` to be sure the value is there.

Reply:
ok 👍